src/lib.rs

```diff
@@ -358,6 +355,10 @@ impl Plugin for EguiPlugin {
         render_graph
             .add_node_edge(base::node::MAIN_PASS, node::EGUI_PASS)
             .unwrap();
+        render_graph
+            // hardcodede until https://github.com/bevyengine/bevy/pull/1464 is merged
+            .add_node_edge("ui_pass", node::EGUI_PASS)
+            .unwrap();
```

Comment:
I'm curious whether this works if a user doesn't have the UI plugin enabled. Probably, we'll have to add this node conditionally based on whether such a node exists or the plugin is enabled. I'm not sure there's even API for checking it though. 🤔

Reply:
Now it only gets added [if the ui pass node exists](https://github.com/mvlabat/bevy_egui/pull/12/commits/caa7eacac1126a9cc670136131a16c7456fdb6ec)